app/demoservice/views.py

```diff
@@ -37,6 +39,24 @@ def _get_github_url(demo):
     return url
 
 
+def _get_launchpad_url(demo):
+    url = "https://code.launchpad.net/~{user}/{repo}/+git/{repo}/+merge/{id}".format(
+        id=demo["github_pr"],
+        repo=demo["github_repo"],
+        user=demo["github_user"]
```

Comment:
Why `github_*` is too deep in the code to have something more generic?

Reply:
Yes, I think refactoring this could do with it's own ticket.